pkg/iac-providers/kubernetes.go

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package iacprovider
+
+import (
+	"reflect"
+
+	k8sv1 "github.com/accurics/terrascan/pkg/iac-providers/kubernetes/v1"
+)
+
+// terraform specific constants
+const (
+	kubernetes   supportedIacType    = "k8s"
+	kubernetesV1 supportedIacVersion = "v1"
```

Comment:
Let's think, if we want default version to be `v1`?

Reply:
Fair point. When adding the version, I was thinking API version, though with Terraform we are using the product version. It's a little tricky even with the API version, since say, Pods are defined in corev1, Deployments may be defined in appsv1beta1, some RBAC constructs I think were only available in alpha, so I just used v1 as a generic v1*. As for using the product version, I'm afraid we might have to update this very frequently if that were the case, since it seems the release cycles move quickly. Any suggestions?

Reply:
going with v1 unless there are any objections